test/numa/tests/memtier/n4c16/test01-only-add-pods/code.var.sh

```diff
@@ -0,0 +1,37 @@
+CPU=1 create guaranteed
+verify \
+    'len(allowed_cpus["pod0"]) == 1'
+report allowed
+
+CPU=2 create guaranteed
+verify \
+    'len(allowed_cpus["pod0"]) == 1' \
+    'len(allowed_cpus["pod1"]) == 2' \
+    'len(allowed_cores["pod1"]) == 1' \
+    'len(allowed_cpus["pod0"].union(allowed_cpus["pod1"])) == 3'
```

Comment:
After having `git rebase HEAD^ --onto master`'d this PR, I think we'll need to replace `allowed_{cpus,threads,cores,nodes}` with `{cpus,threads,cores,nodes}`.